server/shared/types/routes/hicdata.ts

```diff
@@ -1,4 +1,7 @@
 export type HicdataRequest = {
+	/** Value is the 1st parameter of straw tool, and can only use these specific strings but not anything else. */
+	oevalues: 'observed' | 'expected' | 'oe'
```

Comment:
here the value is required but client doesn't send it yet. does it throw any err?

Reply:
@xzhou82 I've tested it on a few files and am not seeing an error. @akanksha-r Did you test and see any error?